# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/validation/encoding/cmds/copyBufferToBuffer/cts.https.html.ini
# Commit: 3f167f8b3137
# Full Hash: 3f167f8b313749f50f135e189e5854bef7ac0c73
# Author: Erich Gubler <erichdongubler@gmail.com>
# Date: 2025-10-03 18:08 +0000
# Regressor Bug: 1991551
# File Overlap Count: 23
# Overlapping Files: third_party/rust/wgpu-hal/Cargo.toml, supply-chain/audits.toml, third_party/rust/naga/src/back/spv/writer.rs, third_party/rust/naga/src/back/msl/writer.rs, third_party/rust/wgpu-types/src/lib.rs
# Description:
#   Bug 1991551 - build(webgpu): update WGPU to 8c4aebc0c1b75e3f7412dfc180745a0717fc30ec r=webgpu-reviewers,supply-chain-reviewers,nical
# ==============================================================================

diff -r 823439f307a0 -r 3f167f8b3137 testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/validation/encoding/cmds/copyBufferToBuffer/cts.https.html.ini
--- a/testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/validation/encoding/cmds/copyBufferToBuffer/cts.https.html.ini	Fri Oct 03 18:01:30 2025 +0000
+++ b/testing/web-platform/mozilla/meta/webgpu/cts/webgpu/api/validation/encoding/cmds/copyBufferToBuffer/cts.https.html.ini	Fri Oct 03 18:08:29 2025 +0000
@@ -3,11 +3,9 @@
 
 
 [cts.https.html?q=webgpu:api,validation,encoding,cmds,copyBufferToBuffer:buffer_state:*]
-  implementation-status: backlog
   [:srcBufferState="destroyed";dstBufferState="destroyed"]
 
   [:srcBufferState="destroyed";dstBufferState="invalid"]
-    expected: FAIL
 
   [:srcBufferState="destroyed";dstBufferState="valid"]
 